ui/src/lib.rs

```diff
@@ -75,6 +77,30 @@ impl Tui {
         let backend = CrosstermBackend::new(stdout);
         let mut terminal = Terminal::new(backend)?;
         terminal.hide_cursor();
+        let pc_ic_maps = known_contracts
```

Comment:
we really need to turn this know_contracts Btreemap thing into a new type -.-

Reply:
yes, or find another way to get rid of it :smile: will be easier when things like #2692 is done so it's easier to reason about, i don't remember other places this is used